apps/extension/src/listeners/message-external.ts

```diff
@@ -0,0 +1,33 @@
+import { assetIdFromBaseDenom } from '@penumbra-zone/wasm/asset';
+
+chrome.runtime.onMessageExternal.addListener((msg, _, respond) => {
```

Comment:
So not as to confuse later devs that stumble upon this file, can you add a code comment above about externally connectable and that this is temporary code?